src/sliding_window_average_by_count.c

```diff
@@ -104,7 +104,7 @@ int sliding_window_average_by_count_add(THANDLE(SLIDING_WINDOW_AVERAGE) handle,
         {
             int window_location = handle->next_available_slot % handle->window_count;
 
-            // Codes_SRS_SLIDING_AVERAGE_WINDOW_45_010: [ If next_available_slot is >= windows_count ]
+            // Codes_SRS_SLIDING_AVERAGE_WINDOW_45_010: [ If next_available_slot is \>= window_count ]
```

Comment:
>\ [](http://example.com/codeflow?start=84&length=1)

This \ should not be in the code. It looks ugly. We need to fix the validate_srs_consistency script to handle characters escaped with \ (like \\, \<, \>)

Reply:
Done